src/molecules/formfields/Select/SimpleSelectField/index.js

```diff
@@ -0,0 +1,102 @@
+import React, { PropTypes } from 'react';
+import classnames from 'classnames';
+
+import styles from 'molecules/formfields/shared/formfields.module.scss';
+import renderPlaceholder from 'utils/Fields/renderPlaceholder';
+
+function SimpleSelectField ( props ) {
+  const {
+    className,
+    type,
+    placeholder,
+    label,
+    selectOptions,
+    forProp,
+    focused,
+    onChange,
+    value
+  } = props;
+
+  return (
+    <div className={ classnames(className, styles.selectfield, { [styles.focused]: focused } ) }>
+      {
+        label &&
+        <label
+          className={ classnames( styles.label, { [styles.focused]: focused } ) }
+          htmlFor={ forProp }
+        >
+          { label }
+        </label>
+      }
+
+      <div className={ classnames( styles.selectbox ) }>
```

Comment:
there are a couple places in this pr where you don't need `classnames` - not a big deal but maybe remove them if they're unnecessary